src/components/dialog/dialog.js

```diff
@@ -619,9 +620,10 @@ function MdDialogProvider($$interimElementProvider) {
         }
         if (options.disableParentScroll) {
           options.restoreScroll();
+          delete options.restoreScroll;
         }
 
-        options.hideBackdrop = null;
+        delete options.hideBackdrop;
```

Comment:
Can you add a comment saying why you are using `delete` instead of setting to null? (since the latter is generally much faster)